src/SourceBuild/content/build.sh

```diff
@@ -259,7 +259,13 @@ if [ "$alternateTarget" == "true" ]; then
   export NUGET_PACKAGES=$NUGET_PACKAGES/smoke-tests
   "$CLI_ROOT/dotnet" msbuild "$SCRIPT_ROOT/build.proj" -bl:"$SCRIPT_ROOT/artifacts/log/Debug/BuildTests_$LogDateStamp.binlog" -flp:"LogFile=$SCRIPT_ROOT/artifacts/logs/BuildTests_$LogDateStamp.log" -clp:v=m ${MSBUILD_ARGUMENTS[@]} "$@"
 else
-  "$CLI_ROOT/dotnet" msbuild "$SCRIPT_ROOT/eng/tools/init-build.proj" -bl:"$SCRIPT_ROOT/artifacts/log/Debug/BuildXPlatTasks_$LogDateStamp.binlog" -flp:LogFile="$SCRIPT_ROOT/artifacts/logs/BuildXPlatTasks_$LogDateStamp.log" -t:PrepareOfflineLocalTools ${MSBUILD_ARGUMENTS[@]} "$@"
+  # BuildXPlatTasks uses NetCurrent but that is not set since Arcade isn't used here.
+  # Bootstrap NetCurrent by deriving it from the installed .NET CLI version.
+  dotnetVersion=$(echo $("$CLI_ROOT/dotnet" --version) | sed -E 's/^([0-9]+\.[0-9]+)\..*/\1/')
+  netCurrent="net$dotnetVersion"
```

Comment:
nit: a bit light-weight variant:
`netCurrent="$(dotnet --version | while IFS='.' read major minor _; do echo "net$major.$minor"; done)"`

Reply:
Thanks! I like the readability of that.